app/src/repository/pokemonMove.js

```diff
@@ -1,11 +1,32 @@
+import DataLoader from "dataloader";
+
 export function PokemonMoveRepository(knex) {
   return {
-    // TODO: Dataloader
-    findPokemonMoveByPokemonIdAndMoveId(pokemonId, moveId) {
-      return knex("pokemon_move").where({
-        pokemon_id: pokemonId,
-        move_id: moveId
-      });
-    }
+    findPokemonMoveByPokemonIdAndMoveId: new DataLoader(
+      ids => {
+        console.log("findPokemonMoveByPokemonIdAndMoveId:", ids);
```

Comment:
This line will need to be removed.

Reply:
As long a we have problems connecting to the database from Zeit I'll leave the logs here, we can remove them later.